src/vs/workbench/api/browser/mainThreadDebugService.ts

```diff
@@ -228,6 +228,7 @@ export class MainThreadDebugService implements MainThreadDebugServiceShape, IDeb
 			parentSession,
 			repl: options.repl,
 			compact: options.compact,
+			simpleUI: options.simpleUI,
```

Comment:
maybe you want to pass the whole `debugUI` structure here. Then it's easier to add more flags while we are discussing the shape of `debugUI`.